core/repos/repo.go

```diff
@@ -113,17 +102,20 @@ func (r *Repo) getCloneParentDirs() []string {
 func (r *Repo) getUrl() string {
 	return "https://" +
 		r.HostingSite + "/" +
-		r.User 		  + "/" +
-		r.RepoName	  + ".git"
+		r.User + "/" +
+		r.RepoName + ".git"
 }
 
 func (r *Repo) getWorkflowDir() string {
-	return r.getCloneDir() + "/workflows/"
+	return filepath.Join(r.getCloneDir(), "workflows")
```

Comment:
Note that here the behavior changes because `filepath.Join` doesn't append a trailing slash. You should check whether `getWorkflowDir()` is called in contexts where a trailing slash is expected.